configure.ac

```diff
@@ -3648,6 +3640,34 @@ if test x$host_win32 = xno; then
 		AC_MSG_RESULT(no)
 	])
 
+	case "$host" in
+		*-*-*freebsd*)
+			dnl *****************************
+			dnl *** Checks for libinotify ***
+			dnl *****************************
+			AC_CHECK_LIB(inotify, inotify_init, LIBS="$LIBS -linotify")
+			if test "x$ac_cv_lib_inotify_inotify_init" = "xyes" ; then
+				AC_DEFINE(HAVE_LIBINOTIFY, 1, [FreeBSD libinotify kqueue shim])
+				dnl Needs to be done this way to avoid collision with various 
+				dnl ports including glib and llvm*
+				METADATA_CFLAGS="-I/usr/local/include"
+			fi
```

Comment:
How is this propagated to metadata/Makefile.am ?

Reply:
Oops, git strikes again; I assumed it would combine both. I just pushed a squashed commit to fix this.
The first commit referenced here also modifies metadata/Makefile.am, but this didn't show in the second commit (which was just to resolve the unsigned wd test.)

Reply:
This still won't work, you'd need to do `AC_SUBST` for it to propagate.

Why not modify `CFLAGS` instead?

Reply:
I'm just as confused as to why it works this way without the `AC_SUBST`. It shouldn't, yet, somehow it does. Best I can figure is that there's some kludge going on to cope with missing SUBSTs. I do agree we should do it the right way, so I'll bump with a quick fix.

The reason to not use `CFLAGS` is multiple collisions that occur due to ports unpredictably. So we need to keep the scope of `-I/usr/local/include` narrower on FreeBSD, particularly since we only really need it in two places; `MONO_NATIVE_CFLAGS` to emulate the CoreFX fix (a la NetBSD,) and metadata.